src/features/auth/components/PasswordSecurityRules.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { render } from '@testing-library/react-native'
+import React from 'react'
+
+import { PasswordSecurityRules } from 'features/auth/components/PasswordSecurityRules'
+
+describe('<PasswordSecurityRules />', () => {
+  it('should display 5 rules', async () => {
+    const { queryByText } = render(<PasswordSecurityRules password={''} />)
+    const twelveCharRule = queryByText('12 Caractères')
+    const capitalRule = queryByText('1 Majuscule')
+    const lowercaseRule = queryByText('1 Minuscule')
+    const numberRule = queryByText('1 Chiffre')
+    const specialCharRule = queryByText('1 Caractère spécial (%,#@!)')
+
+    expect(twelveCharRule).toBeTruthy()
```

Comment:
Si tu veux vraiment enlever des lignes de code :
```suggestion
    expect(queryByText('12 Caractères')).toBeTruthy()
```
À toi de voir.